meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mmevk/full_image.uuu.in

```diff
@@ -1,24 +1,15 @@
 uuu_version 1.2.39
 
-SDP: boot -f imx-boot-mfgtool
+SDP: boot -f SPL-mfgtool
 
-# These commands will be run when use SPL and will be skipped if no spl
-# SDPU will be deprecated. please use SDPV instead of SDPU
-SDPU: delay 1000
-SDPU: write -f imx-boot-mfgtool -offset 0x57c00
-SDPU: jump
-
-# These commands will be run when use SPL and will be skipped if no spl
-# if (SPL support SDPV)
 SDPV: delay 1000
-SDPV: write -f imx-boot-mfgtool -skipspl
+SDPV: write -f u-boot-mfgtool.itb
 SDPV: jump
 
 FB: ucmd setenv fastboot_dev mmc
 FB: ucmd setenv mmcdev ${emmc_dev}
 FB: ucmd mmc dev ${emmc_dev}
 FB: flash -raw2sparse all ../@@MFGTOOL_FLASH_IMAGE@@-@@MACHINE@@.wic
-FB: flash bootloader ../imx-boot-@@MACHINE@@
```

Comment:
We still need to flash bootloaders in the full_image.uuu file.

Reply:
yes, in case we use boot0 parition.
If you check wks file for the wic image creation, you'll see that in the current setup I used user hw partition for everything, including SPL and U-Boot FIT